lib/cog/command/pipeline/executor.ex

```diff
@@ -660,4 +662,8 @@ defmodule Cog.Command.Pipeline.Executor do
       end
     end
   end
+
+  defp services_root do
+    ServiceEndpoint.url <> "/v1/services"
```

Comment:
Instead of pulling the root out of the request, shouldn't the commands specify which endpoint to use? At least in regards to the version. Otherwise wouldn't all the commands break when we add '/v2/services`? I assume at some point we could have commands using different versions of the services endpoint.